tests/execute/restraint/report_result/test.sh

```diff
@@ -0,0 +1,22 @@
+#!/bin/bash
+. /usr/share/beakerlib/beakerlib.sh || exit 1
+
+rlJournalStart
+    rlPhaseStartSetup
+        rlRun "output=\$(mktemp)" 0 "Create output file"
+        rlRun "tmp=\$(mktemp -d)" 0 "Create tmp directory"
+        rlRun "set -o pipefail"
+        rlRun "pushd data"
+    rlPhaseEnd
+
+    rlPhaseStartTest 'Verify mocked Restraint rstrnt-report-result file generated correctly and read by tmt.'
+        rlRun "tmt run -vvvddd --remove | tee $output"
```

Comment:
I'd suggest to add an `rlAssertGrep` check for good/bad/skip tests and verify expected values:

    pass /smoke/good
    fail /smoke/bad
    info /smoke/skip

What do you think about mapping skipped test to an `info` value? I'd say it would better match the fact that the test was skipped. See also the [execute](file:///home/psss/git/tmt/docs/_build/html/spec/plans.html#execute) section for detailed description of supported test result status.

Reply:
That makes sense, so I've added the mapping in my most recent commit.